notificationbox/implementation.js

```diff
@@ -57,7 +59,10 @@ class Notification {
       }
     };
 
-    this.getNotificationBox().appendNotification(this.options.label, this.notificationId, this.imageURL, this.options.priority, buttons, callback);
+    let element = this.getNotificationBox().appendNotification(options.label, notificationId, imageURL, options.priority, buttons, callback);
+    if (options.css) {
+      element.setAttribute("style", element.getAttribute("style") + options.css);
```

Comment:
```suggestion
      element.setAttribute("style", element.getAttribute("style") + "; " + options.css + "; ");
```

Just a suggestion, not mandatory.

Reply:
Could we just have a color pair ({ backgroundColor, textColor} ) as input and just set 
```
element.style["background-color"] = backgroundColor;
element.style["color"] = textColor;
```

Would this not be more WebExtension API - ish?

Reply:
OK I've rewritten the code to that effect and rebased it due to the merge conflict.